internal/cli/atlas/privateendpoints/gcp/interfaces/interfaces.go

```diff
@@ -29,6 +29,7 @@ func Builder() *cobra.Command {
 		Short:   "Manage Atlas GCP private endpoint interfaces.",
 	}
 	cmd.AddCommand(DescribeBuilder())
+	cmd.AddCommand(DeleteBuilder())
```

Comment:
```suggestion
	cmd.AddCommand(DescribeBuilder(), DeleteBuilder())
```

Reply:
Thanks Gustavo - I have changed that in the latest commit.